sdcm/utils/udf_scripts/xwasm_plus_acc.yaml

```diff
@@ -0,0 +1,151 @@
+# UDF script: returns a tuple with the accumulator incremented by arg and counter incremented by 1
```

Comment:
Agreed. I'm considering removing this function (and the final div function in xwasm as well), since they don't perform tuple deserialization like they should, so no changes to this one atm, but I'll be adding the source code and compile commands to other added scripts.

Reply:
@KnifeyMoloko maybe what we should do in SCT is to have the C++ code and the SCT will be responsible of the compilation + translation to wat format.
So, we can change the human readable code only.